global.json

```diff
@@ -24,7 +24,7 @@
     "xcopy-msbuild": "17.1.0"
   },
   "native-tools": {
-    "jdk": "11"
+    "jdk": "11.0.24"
```

Comment:
@wtgodbe this change broke the entire VMR now that images got updated to a newer version of the jdk. This already happened once in the past which is why we made this change: https://github.com/dotnet/aspnetcore/commit/4a3afe3b23e062ca120c08a2761b3ea7775a03f9

Can you please describe why the minor and patch versions were necessary to be added here?

cc @mmitche

Reply:
Arcade offers scripts to install native tools which support defining just a major version or even "latest": https://github.com/dotnet/aspnetcore/blob/main/eng/common/init-tools-native.ps1

Any reason why you can't use those? The problem with the hardcoded version is that it breaks whenever the CI images get updated. I.e. right now all aspnetcore builds should be broken. See https://github.com/dotnet/sdk/pull/45822

Reply:
I don't see any reason why we can't - I'd actually prefer it. Do we just need to add a pre-step in CI that runs the script?

Reply:
We used to install the JDK manually ourselves w/ https://github.com/dotnet/aspnetcore/blob/main/eng/scripts/InstallJdk.ps1, but stopped with https://github.com/dotnet/aspnetcore/pull/54421. I was under the impression we had to use whatever was pre-installed on the build machine/in the VMR tool cache

Reply:
Let's move that discussion to https://github.com/dotnet/source-build/issues/4816